plugin/OmniSharp.vim

```diff
@@ -124,5 +124,8 @@ if g:OmniSharp_selector_ui ==? 'ctrlp'
     let g:ctrlp_extensions += ['findsymbols', 'findcodeactions']
   endif
 endif
+if g:OmniSharp_selector_ui ==? 'fzf'
+  let g:OmniSharp_fzf_options = get(g:, 'OmniSharp_fzf_options', {})
+endif
```

Comment:
Using the `get()` construct as described above, we don't need to initialise the `g:OmniSharp_fzf_options` dict here